indexer/indexer_tests/drules_selector_parser_test.cpp

```diff
@@ -16,6 +16,16 @@ UNIT_TEST(TestDruleSelectorIsSet)
   TEST_EQUAL(SelectorOperatorIsSet, e.m_operator, ());
 }
 
+UNIT_TEST(TestDruleSelectorIsSet2)
+{
+  SelectorExpression e;
+  TEST(ParseSelector("envelope_area", e), ());
```

Comment:
limit_rect does not reflect that it is area (sq.m.), besides, postgis use the same term, so, it is not so mistery :)